docs/basics/janusgraph-cfg.md

```diff
@@ -447,7 +448,7 @@ HBase storage options
 | storage.hbase.short-cf-names | Whether to shorten the names of JanusGraph's column families to one-character mnemonics to conserve storage space | Boolean | true | FIXED |
 | storage.hbase.skip-schema-check | Assume that JanusGraph's HBase table and column families already exist. When this is true, JanusGraph will not check for the existence of its table/CFs, nor will it attempt to create them under any circumstances.  This is useful when running JanusGraph without HBase admin privileges. | Boolean | false | MASKABLE |
 | storage.hbase.snapshot-name | The name of an existing HBase snapshot to be used by HBaseSnapshotInputFormat | String | janusgraph-snapshot | LOCAL |
-| storage.hbase.snapshot-restore-dir | The temporary directory to be used by HBaseSnapshotInputFormat to restore a snapshot. This directory should be on the same File System as the HBase root dir. | String | /tmp | LOCAL |
+| storage.hbase.snapshot-restore-dir | The temporary directory to be used by HBaseSnapshotInputFormat to restore a snapshot. This directory should be on the same File System as the HBase root dir. | String | /var/folders/dw/4h320h7d5jbfyqhtc_ghw2_jqhk0c8/T/ | LOCAL |
```

Comment:
We should not change this file directly. To be consistent, execute:
```
mvn --quiet clean install -DskipTests=true -pl janusgraph-doc -am
```
After the execution:
```
git add -A
git commit --amend -s
git push -f
```

Reply:
still the same, I think this is because I add a property in the graph configuration.

Reply:
TBH every time when I do a maven install, a similar diff is generated for me as well. Not sure if this is OS-related (I am using Mac OS btw). Anyway, @9lan9 you can manually revert this change, and then CI shall pass.

Reply:
@9lan9 Just fix it manually. If you like you can open up an issue.

Reply:
> TBH every time when I do a maven install, a similar diff is generated for me as well. Not sure if this is OS-related (I am using Mac OS btw). Anyway, @9lan9 you can manually revert this change, and then CI shall pass.

ok, thanks! will fix this one manually.

Reply:
> @9lan9 Just fix it manually. If you like you can open up an issue.

ok, thanks!